docs/usage/automerge-configuration.md

```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
 ### Codeowners
 
 Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.
+
+### Overriding global automerge
+
+You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
```

Comment:
Shouldn't we tell this shared preset? Not all people are using `.github` repo for this. 😏

Reply:
I'm sorry, I really don't understand what you mean here. Can you try explaining again, but with different words?

Reply:
I think we've done enough describing already. There's nothing about this overriding which is specific to automerge anyway.

Reply:
```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
It does not matter where you've put the global config, the important point in this example is that you're extending from a global config that's somewhere else.
For this example we'll assume you put your config in a repository on GitHub, called `.github`.
```

How about this then? It explains a bit more, and tells readers that the important point is that you're extending from a config that's _somewhere else_.